xbmc/cores/AudioEngine/AESinkFactory.cpp

```diff
@@ -35,11 +35,12 @@
 #elif defined(TARGET_LINUX) || defined(TARGET_FREEBSD)
   #if defined(HAS_ALSA)
     #include "Sinks/AESinkALSA.h"
+  #else
```

Comment:
OK to change the `else` to `#elif defined(TARGET_FREEBSD)` here and on other positions (to have OSS only on FreeBSD)?

If it becomes separated to a own `#elif defined(TARGET_FREEBSD)` must be the PulseAudio part inserted two times (also available on FreeBSD).